js/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { PublicKey } from '@solana/web3.js';
+import { ProgramAddresses } from './types';
+
+export const MAINNET_PROGRAM_ADDRESSES: ProgramAddresses = {
+  solidoProgramId: new PublicKey(
+    'CrX7kMhLC3cSsXJdT7JDgqrRVWGnUpX3gfEfxxU2NVLi'
+  ),
+  solidoInstanceId: new PublicKey(
+    '49Yi1TKkNyYjPAFdR9LBvoHcUjuPX4Df5T5yv39w2XTn'
+  ),
+  stSolMintAddress: new PublicKey(
+    '7dHbWXmci3dT8UFYWYZweBLXgycu7Y3iL6trKn1Y7ARj'
+  ),
+};
```

Comment:
:+1: I cross-checked these addresses against https://docs.solana.lido.fi/deployments and I can confirm that they are correct.